scripts/build/download-tool.sh

```diff
@@ -78,7 +78,8 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+echo "Downloading $ARGV_URL"
+curl --continue-at - --retry 100 --location --output "$TEMP_OUTPUT" "$ARGV_URL"
```

Comment:
I don't think we need `--continue-at` and `--retry`?
And as we're using `-L` in `electron-download-package.sh`, perhaps we should switch to the short-option flags `-L -o` instead of `--location --output`? (either that, or update `electron-download-package.sh` to match here)